### PR TITLE
Even more fixes, also more admin logs

### LINKED ISSUE #6511: Vampiric death touch is not logged
Feature request.  Or maybe if you really stretch it, oversight.

The other day someone got deathtouched and I couldn't find it in the logs.  It's something that would be really nice to have.  It can and probably will kill another player and there's no indication in the logs that it ever happened which can lead to confusion.

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #6511

Vampire touch is now logged. Admins are also notified by it, and by enthralling (with a button to jump to the victim's location)

Added many logs so admins learn right away which mode it is at round start, and for when/why a mode fails to set-up. (So now we'll now which modes always fail to start and why)

### CODE REVIEW

File: code/game/gamemodes/revolution/revolution.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@
 	var/head_check = 0
 	for(var/mob/new_player/player in player_list)
 		if(player.mind.assigned_role in command_positions)
-			head_check = 1
+			head_check++
```
Comment (Probe1) [inline code/game/gamemodes/revolution/revolution.dm]: What difference will this make?

Comment (DeityLink) [inline code/game/gamemodes/revolution/revolution.dm]: Read the new logs a few lines down.

Comment (Probe1) [inline code/game/gamemodes/revolution/revolution.dm]: I didn't understand.

**Comment by DeityLink** [comment]
By the way travis is full of shit the build is fine.

**Comment by PJB3005** [comment]
Can confirm travis fucked up.

**Pull request merged**